### PR TITLE
Ruby branch

### PR DESCRIPTION
**Pull request by husseinmaad**
@mechamanny

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -1,39 +1,60 @@
...
-puts "Would you like to enroll in the company’s health insurance (y/n)?"
-health_ins = boolean(gets.chomp)
+puts "How many employees you will be process?"
+num_emp_process = gets.chomp.to_i
+
+while num_emp_process > 0 do
+  puts "Please start enter the employee information:"
+  
+  # this method to convert the string to boolean
+  def boolean(string)
```
Comment (mechamanny) [inline ruby/vampires.rb:9]: Avoid defining methods inside a loop, they get redefined every time the loop runs. Methods only need to be defined once.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
+    return (string.downcase == "y" ? true : false)
+  end
+
+  # verify the age method 
+  def age_verify(age, year)
+    return (Time.new.year - year == age ? true : false)
```
Comment (mechamanny) [inline ruby/vampires.rb:15]: Ternary is unnecessary here, `Time.new.year - year == age` already returns a true or false value.

File: ruby/vampires.rb [hunk 3]
```diff
@@ -1,39 +1,60 @@
...
+  end 
+# this will ask the user some questions and apply cretin check conditions
+  puts "Whats your name?"
+  emp_name = gets.chomp.to_s
```
Comment (mechamanny) [inline ruby/vampires.rb:19]: `gets` is already a string, no need to `to_s`

**Comment by mechamanny** [comment]
@husseinmaad Some good components here, just need to refine the execution.

**Comment by husseinmaad** [comment]
Hey Emmanuel,
Thanks for your advises I will refine the execution.
Thank you,
On Dec 15, 2016, at 6:50 AM, Emmanuel Kaunitz <notifications@github.com<mailto:notifications@github.com>> wrote:


@husseinmaad<https://github.com/husseinmaad> Some good components here, just need to refine the execution.

**Pull request merged**